findspam.py

```diff
@@ -1095,6 +1120,13 @@ class FindSpam:
          'max_rep': 20, 'max_score': 0},
     ]
 
+    # Toxic content using Perspective
+    if GlobalVars.perspective_key:  # don't bother if we don't have a key, since it's expensive
```

Comment:
What does"rather expensive" mean? Are we going to have performance issues?

Reply:
No. It's one POST, which is expensive compared to the other things we do but it's not that slow. There's just no point in running it if we don't have the Perspective key in the config (it's a secret, so it might not be there if someone is say running a test instance).